hindi/randeep_hooda/randeep.md

```diff
@@ -0,0 +1,3 @@
+Randeep Hooda (pronounced [r?d?i?p hu??a?]; born 20 August 1976) is an Indian film actor who appears in Hindi-language films. Hooda began acting in school productions at a young age. After completing his studies in Melbourne, Australia, he returned to India and began modelling and acting in theatre.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.